db/data/20240722092527_add_organization_id_to_requests.rb

```diff
@@ -3,7 +3,7 @@
 class AddOrganizationIdToRequests < ActiveRecord::Migration[6.1]
   def up
     ActiveRecord::Base.transaction do
-      organization = Organization.singleton
+      organization = Organization.first
```

Comment:
what happens later on instances when there are multiple organizations? I would even add sth like return if Organization.count != 1 to ensure that it is only run with exactly one organization